setup.py

```diff
@@ -0,0 +1,16 @@
+import setuptools
+
+
+setuptools.setup(
+    name="ts_IntegrationTests",
+    description="TSSW integration test scripts.",
+    version="0.0.1",
```

Comment:
... instead of hard-coding the version here do:

```
    use_scm_version={
        "write_to": "python/lsst/ts/IntegrationTests/version.py",
        "write_to_template": scm_version_template,
    },
```